### PR TITLE
Add OAuth Playground Docs

### PR DESCRIPTION
**Pull request by sherenewso2**
## Purpose
- address review comments for authentication with OIDC doc
- Add OAuth Playground docs and navigation

### CODE REVIEW

File: en/docs/samples/auth-code-playground.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+
+    - **Access Token Endpoint:** https://localhost:9443/oauth2/token 
+
+    - **Client Secret:** The client secret received when registering the service provider.
+
+	- **PKCE Verifier:** This will be populated using the value generated in step 1 only if you are using PKCE. 
+      
+
+5. Click **Get Access Token**. At this point, the application receives the access token.
```
Comment (DMHP) [inline en/docs/samples/auth-code-playground.md]: These samples are common for both OAuth2.0 and OIDC right? So shall we include the steps of invoking the user info and session management details to the same sample? Maybe we can include a different page for those.

File: en/docs/samples/auth-code-playground.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Authorization Code Grant with OAuth Playground
+
+This page guides you through using a **sample Playground application** to try out authentication to an OAuth/OpenID Connect web application using the [Authorization Code](insertlink) grant type.
```
Comment (DMHP) [inline en/docs/samples/auth-code-playground.md]: OAuth >> OAuth2.0

File: en/docs/samples/auth-code-playground.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+    - **Authorization Grant Type:** Authorization Code 
+    
+    - **Client ID:** The OAuth Client Key recieved when registering the service provider.
+    
+    - **Callback URL:** http://wso2is.local:8080/playground2/oauth2client
+    
+    - **Authorize Endpoint:** https://localhost:9443/oauth2/authorize
+    
+    - **Scope:** Any scope you wish to obtain the token for (e.g., `openid`). This field is optional.
```
Comment (DMHP) [inline en/docs/samples/auth-code-playground.md]: Shall we specially mention that if someone needs to utilize the sample with OIDC they have to use 'openid' scope mandatorily?

File: en/mkdocs.yml [hunk 4]
```diff
@@ -126,6 +126,11 @@ nav:
       - 'Java': samples/java.md
       - 'NodeJs': samples/nodejs.md
       - 'OpenID Connect': samples/regular-webapp-oidc-sample.md
+  - 'OAuth Playground':
```
Comment (DMHP) [inline en/mkdocs.yml]: OAuth >> OAuth2.0/OIDC

File: en/docs/samples/oauth-playground.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+	`                    http://wso2is.local:8080/playground2/                   ` .
+
+	By default Tomcat runs on port 8080. If you have configured
+	it to run on a different port make sure to update the URL
+	and then access the playground application.  
+
+6. You are directed to the landing page of the sample application. Click on **Import Photos** and the following
+page appears.  
+
+	![playground-app.png](../assets/img/guides/playground-app.png)
```
Comment (ayshsandu) [inline en/docs/samples/oauth-playground.md]: Make these images zoomable is applicable.

File: en/docs/samples/auth-code-playground.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/samples/auth-code-playground.md]: ack, will send a new PR with this info

**Pull request merged**